service/src/main/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiController.java

```diff
@@ -510,20 +523,30 @@ private ApiDeleteControlledGcpAiNotebookInstanceResult fetchNotebookInstanceDele
   public ResponseEntity<ApiCloneControlledGcpBigQueryDatasetResult> cloneBigQueryDataset(
       UUID workspaceId, UUID resourceId, @Valid ApiCloneControlledGcpBigQueryDatasetRequest body) {
     final AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
+    final AuthenticatedUserRequest petRequest =
```

Comment:
Maybe make this a little method in the controller?
The cut'n'paste is used many times. I don't think we ever need a userRequest except as input to the getPetCredentials.